server/app/routes/v1/audit_logs_api.rb

```diff
@@ -0,0 +1,21 @@
+module V1
+  class AuditLogsApi < Roda
+    include TokenAuthenticationHelper
+    include CurrentUser
+    include RequestHelpers
+    include Auditor
+
+    plugin :streaming
+    
+    route do |r|
+      r.get do
+        halt(403, 'Access denied') unless current_user.master_admin?
+
+        limit = request.params['limit'] || 500
```

Comment:
The thing from container logs that @SpComb made should probably be integrated into the audit logs too. With tailing.